src/index.js

```diff
@@ -1,8 +1,8 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import './index.css';
-import App from './App';
-import * as serviceWorker from './serviceWorker';
+import './App/index.css';
+import App from './App/App.js';
+import * as serviceWorker from './App/serviceWorker';
```

Comment:
Please don't import all like that, just import what do you use so that the bundle file won't contain redundant code.

Reply:
These were imported when i used npm-create-app, im unsure what is needed and what is not. Can you advise further?

Reply:
Ok, I got it, so let these lines as the origin. However please remember we shouldn't make a copy of that for our codes